src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPViewBase.h

```diff
@@ -502,6 +502,8 @@ protected slots:
       String in;
       String out;
       String file_name;
+      String file_extension;
```

Comment:
it may be better to use
```
String file_name_in;  ///< temporary filename including extension as input to TOPP tool
String file_name_out; ///< temporary filename including extension written to by TOPP tool
```

Reply:
Good idea, that is much cleaner